src/frontend/src/invites.tsx

```diff
@@ -17,6 +35,56 @@ export const Invites = () => {
         loadInvites();
     }, []);
 
+    const update = async (code: string) => {
+        const updatedInvite = updatedInvites.find(
+            ({ code: inviteCode }) => code === inviteCode,
+        );
+        if (!updatedInvite) {
+            return;
+        }
+        return window.api
+            .call<any>(
+                "update_invite",
+                code,
+                updatedInvite.credits !== undefined &&
+                    updatedInvite.credits >= 0
+                    ? updatedInvite.credits
+                    : null,
+                updatedInvite.realm_id !== undefined
+                    ? updatedInvite.realm_id
+                    : null,
+            )
+            .then(async (response) => {
+                if ("Err" in (response || {})) {
+                    alert(`Error: ${response.Err}`);
+                    setBusy(true);
+                    await loadInvites(); // Set back to prior state
+                    setBusy(false);
```

Comment:
Why do we need this? The state in UI should not have changed during the call, right?

Reply:
This is to reset to previous state if update fails.